config/application.yml.default.docker

```diff
@@ -14,6 +14,8 @@ production:
   database_worker_jobs_host: ['env', 'POSTGRES_WORKER_HOST']
   database_worker_jobs_password: ['env', 'POSTGRES_WORKER_PASSWORD']
   database_worker_jobs_sslmode: ['env', 'POSTGRES_WORKER_SSLMODE']
+  database_primary_advisory_locks: false
+  database_primary_prepared_statements: false
```

Comment:
this needs to be in the actual configs and i don't know how to do that, how do you do that?